scripts/hostcfgd

```diff
@@ -160,7 +162,7 @@ class Feature(object):
         self.name = feature_name
         self.state = self._get_feature_table_key_render_value(feature_cfg.get('state'), device_config or {}, ['enabled', 'disabled', 'always_enabled', 'always_disabled'])
         self.auto_restart = feature_cfg.get('auto_restart', 'disabled')
-        self.has_timer = safe_eval(feature_cfg.get('has_timer', 'False'))
+        self.delayed = safe_eval(feature_cfg.get('delayed', 'False'))
```

Comment:
>delayed [](http://example.com/codeflow?start=50&length=7)

The ConfigDB schema change it not backward compatible. Could you support both schema in this daemon? Or, could you check the schema version and fail immediately if old version?

Reply:
Added the check as you requested